frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.jsx

```diff
@@ -113,6 +111,33 @@ export function DashCardCardParameterMapper({
   const virtualCardType = getVirtualCardType(dashcard);
   const isNative = isNativeDashCard(dashcard);
 
+  let selectedMappingOption;
+  if (isVirtual || isAction || isNative) {
+    selectedMappingOption = target
+      ? mappingOptions.find(option =>
+          _.isEqual(normalize(option.target), normalize(target)),
+        )
+      : undefined;
+  } else {
+    const columns = Lib.visibleColumns(question.query(), -1);
```

Comment:
do we need visible or filterable columns here? or doesn't matter? /cc @bshepherdson

Reply:
Short answer: yes, `visibleColumns` is the right thing here.

Long answer: `filterableColumns` returns a subset of the `visibleColumns`. Usually it's actually the same set, since every column is filterable in principle even if some cases are clumsy (eg. string `=` on a JSON column). The main reason to call `filterableColumns` is that it includes a bunch of extra information about what filter operators can be applied to each column. For the purposes of this matching, that extra work is wasted.

(The other `___able-columns` often return smaller subsets, since not every column is orderable, joinable, etc. They also include some extra information, eg. whether the column is already used in an `ORDER BY` on this query.)